zipkin-ui/test/component_data/default.test.js

```diff
@@ -0,0 +1,32 @@
+import {
+  parseApiQuery
+} from '../../js/component_data/default';
+
+chai.config.truncateThreshold = 0;
+
+describe('parseApiQuery', () => {
+  const should = require('chai').should();
```

Comment:
well it hates things that have no return value. Ex. `expect(undefined).to.be.undefined;` results in `Expected an assignment or function call and instead saw an expression`